contrib/release/start-release.sh

```diff
@@ -107,6 +107,10 @@ main() {
     fi
 
     target_branch=$(echo "$version" | sed "$BRANCH_REGEX")
+    if ! git ls-remote --exit-code --heads $REMOTE $target_branch; then
+        target_branch=$(echo "$old_version" | sed "$BRANCH_REGEX")
+        old_branch="$target_branch"
```

Comment:
@aanm you might want to take a closer look at this second commit. I added this, but I'm hitting this odd issue with the release notes tool. Not quite sure what's going on, but I can revert this line out if you think it doesn't belong.

cf. https://github.com/cilium/release/issues/96